### ISSUE TITLE
ERROR: Input.script filter -- any idea on how to fix this issue?

### ISSUE THREAD

**Issue by WhatsNewSaes** [johnthepink/alfred2-basecamp#10]
[2018-10-31 13:06:30][ERROR: input.scriptfilter] Code 1: /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- alfred (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from ./main.rb:8:in `<main>'

**Comment by lachieh** [comment]
This is a result of High Sierra using Ruby 2.3.0 instead of 1.8. I fixed this by simply renaming the `bundle/ruby/2.0.0` folder inside of the workflow to `bundle/ruby/2.3.0` and it works again.

**Comment by gg67** [comment]
Took me a while to find the right place for this fix! For future reference

1. Open the workflow via from here
![image](https://user-images.githubusercontent.com/1463454/87364881-5d298a80-c53a-11ea-8121-685b65496647.png)
2. Rename the folder 2.0.0 to 2.3.0 as mentioned above
![image](https://user-images.githubusercontent.com/1463454/87364922-7cc0b300-c53a-11ea-95b4-1808a62748ef.png)

**Comment by johnthepink** [comment]
The latest version published (2.1.2) supports the default ruby version on OSX Catalina, which is 2.6. So @gg67's comment may apply if your ruby version is different.

If you experience this issue, you need to change the folder name to the minor version of ruby that you are running. (2.0.0, 2.1.0, 2.3.0, 2.6.0, etc).

To find your ruby version, open the "Terminal" app, and issue this command: `ruby --version`. You will see something like this:

```
ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
```

This means you are running 2.6.3, change to the minor version which is 2.6.0, that is what the folder name should be.

**Comment by johnthepink** [comment]
I now this isn't ideal, and I have a few ideas for making this better. Hopefully will have an update soon.

**Comment by johnthepink** [comment]
Hopefully this is fixed by 2.1.5.
